packages/osl-management-console-image/generated/modules/osl/management-console/configure-httpd.sh

```diff
@@ -44,4 +44,9 @@ cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
 chmod +x "${MGMT_CONSOLE_HOME}/launch/entrypoint.sh" "${MGMT_CONSOLE_HOME}/image-env-to-json-standalone"
 chown -R "${USER_ID}" "${MGMT_CONSOLE_HOME}"
 
+
+# Fixing /var/www permissions
+chgrp -R 0 ${HTTPD_LOG_PATH} ${HTTPD_VAR_RUN} ${HTTPD_DATA_PATH}/html /management-console \
```

Comment:
The generated is different from the sources, isn't it?
I guess you need need to update it and push again.

Reply:
@rgdoliveira thank you, updated now